api/indigo.h

```diff
@@ -807,6 +811,7 @@ CEXPORT int indigoIterateRDF(int reader);
 CEXPORT int indigoIterateSmiles(int reader);
 CEXPORT int indigoIterateCML(int reader);
 CEXPORT int indigoIterateCDX(int reader);
+CEXPORT int indigoIterateJSON(int reader);
```

Comment:
I can't find definition of this function.

Reply:
fixed